examples/traefik/docker-compose.yml

```diff
@@ -140,11 +146,12 @@ services:
     # Video bridge
     jvb:
         image: jitsi/jvb
+        restart: ${RESTART_POLICY}
         ports:
             - '${JVB_PORT}:${JVB_PORT}/udp'
-            - '${JVB_TCP_PORT}:${JVB_TCP_PORT}'
```

Comment:
you want both

Reply:
Could you please elaborate what you mean? I do not see what you mean. This is from the current `docker-compose.yml` I got from git:
```yml
        ports:
            - '${JVB_PORT}:${JVB_PORT}/udp'
            - '${JVB_TCP_MAPPED_PORT}:${JVB_TCP_PORT}'
```

Reply:
Doh, sorry, my bad...